src/components/lists/CheckedBeforeShipmentWarehouseConsignments.tsx

```diff
@@ -87,6 +103,12 @@ function CheckedWarehouseConsignments() {
   return (
     <>
       <Container maxWidth="xl" sx={mainContainerStyle}>
+        <Grid container spacing={2}>
+          <Grid item xs={8}></Grid>
```

Comment:
U need do like this
![image](https://user-images.githubusercontent.com/60444425/174286286-79d6e03b-c00e-4af6-8be4-3ed33c813b2b.png)